lib/matplotlib/backends/backend_svg.py

```diff
@@ -26,196 +26,120 @@
 from matplotlib.transforms import Affine2D, Affine2DBase
 from matplotlib import _png
 
-from xml.sax.saxutils import escape as escape_xml_text
+from xml.etree import ElementTree as etree
 
 backend_version = __version__
 
-# ----------------------------------------------------------------------
-# SimpleXMLWriter class
```

Comment:
@mdboom I'm trying to look at this again.  My thought is to modify the XMLWriter class with an argument on the __init__ method to give an option to write out the tree immediately rather than build a tree in memory.  I can then write out images without building the tree, but if an animation is being made, build the tree in memory.